examples/nextjs/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["next/babel"],
+  "plugins": [["styled-components", { "ssr": true }]]
```

Comment:
I would also add `"pure": true` to the config here so more dead code can be eliminated. See https://styled-components.com/docs/tooling#dead-code-elimination